metricbeat/helper/interfacer.go

```diff
@@ -32,6 +33,7 @@ type ModuleConfig struct {
 	MetricSets []string `config:"metricsets"`
 	Enabled    bool     `config:"enabled"`
 	Selectors  []string `config:"selectors"`
+	Filter     []filter.FilterConfig
```

Comment:
It's the same for meta fields. They can be applied on a prospector level in filebeat, but also on the top level.